src/SFA.DAS.AdminService.Web/Infrastructure/RoatpApplyTokenService.cs

```diff
@@ -1,28 +1,29 @@
 ﻿using Microsoft.Azure.Services.AppAuthentication;
 using Microsoft.Extensions.Hosting;
 using SFA.DAS.AdminService.Settings;
+using System.Threading.Tasks;
 
 namespace SFA.DAS.AdminService.Web.Infrastructure
 {
     public class RoatpApplyTokenService : IRoatpApplyTokenService
```

Comment:
@ChrisJWoodcock Should this also be upgraded to use Azure Identity, or can we leave it as it is?

Reply:
We can leave it as-is - otherwise we would have to test it and it exists for the other use of the Admin Service to the new Roatp V2 Service

Reply:
So technically by updating to use the async version of the GetToken in the base interface - you have been forced to update the other client(s) adding the await keyword before the calls - question is: Is a regression test good enough to cover that - and that is not code our team maintains...